data/nlu/nlu.md

```diff
@@ -4421,6 +4441,11 @@
 - you will know it from the single red rose it carries between its teeth
 - you're a woman
 - ı am learning python
+- CALL THE POLICE
+- I want to speak binary with you
```

Comment:
should this be in `ask_languagesbot`? Would kindof be a perfect response haha

Reply:
Yes 👍